ansible/doc/README.testbed.Internal.md

```diff
@@ -0,0 +1,129 @@
+# Testbed internals
+
+## Topology definition
+
+ - List of currently defined topologies in veos inventory file
+```
+[servers:vars]
+topologies=[‘t1', ‘t1-lag', 't0', 'ptf32', 'ptf64']
```

Comment:
the main change is to split the document into several sections. will address new topology later.